src/quickgui/qgsquickpositionkit.cpp

```diff
@@ -24,10 +24,24 @@
 QgsQuickPositionKit::QgsQuickPositionKit( QObject *parent )
   : QObject( parent )
   , mAccuracy( -1 )
+  , mAccuracyUnits( QgsUnitTypes::toAbbreviatedString( QgsUnitTypes::DistanceMeters ) )
   , mDirection( -1 )
   , mHasPosition( false )
   , mIsSimulated( false )
 {
+
+  mCoordinateTransformer.reset( new QgsQuickCoordinateTransformer() );
```

Comment:
Why it needs to be pointer at all? could be simple instance? Can we use QgsCoordinateTransformer directly (no need to use helper created just for QML)